webpack.config.js

```diff
@@ -10,6 +10,12 @@ module.exports = {
     path: path.join(__dirname, '/dist'),
     filename: 'index_bundle.js'
   },
+  resolve: {
+    // force usage of my luma gl version
```

Comment:
This is new to me. How is this different than pinning version.

Reply:
Vestigial.